File: Plugins/Aspose_Slides_Java_for_PHP/src/aspose/slides/WorkingWithPresentation/OdpToPptx.php
<?php

namespace Aspose\Slides\WorkingWithPresentation;

use com\aspose\slides\Presentation as Presentation;
use com\aspose\slides\SaveFormat as SaveFormat;

class OdpToPptx {

    public static function run($dataDir=null)
    {
        # Instantiate a Presentation object that represents a PPTX file
        $pres = new Presentation($dataDir . "Source.odp");

        # Saving the PPTX presentation to PPTX format
        $save_format = new SaveFormat();
        $pres->save($dataDir . "Source.pptx", $save_format->Pptx);

        print "Document has been converted, please check the output file.";
    }

}

File: Plugins/Aspose-Slides-Java-for-Jython/asposeslides/WorkingWithPresentation/OdpToPptx.py
from asposeslides import Settings
from com.aspose.slides import Presentation
from com.aspose.slides import SaveFormat
from com.aspose.slides import XpsOptions

class OdpToPptx:

    def __init__(self):
                        
         
        dataDir = Settings.dataDir + 'WorkingWithPresentation/OdpToPptx/'
        
        # Instantiate a Presentation object that represents a PPTX file
        pres = Presentation(dataDir + "Source.odp")

        # Saving the PPTX presentation to PPTX format
        save_format = SaveFormat
        pres.save(dataDir + "Source.pptx", save_format.Pptx)

        print "Document has been converted, please check the output file."


if __name__ == '__main__':        
    OdpToPptx()